### PR TITLE
move scripts/magic_folder_cli to magic_folder/cli

### PR DESCRIPTION
**Pull request by meejah**
started a bit of re-org while getting up to speed on "stand-alone magic-folder". It did occur to me that maybe re-orgs are disruptive ... but this one is at least "just one file"...? (Anyway, we could hold off on this if that seems like a good idea).

https://github.com/LeastAuthority/magic-folder/issues/6

### CODE REVIEW

**Comment by meejah** [comment]
@exarkun this could be merged -- unless you think its better to do all re-org at once?

**Comment by exarkun** [comment]
Incremental improvement is great.   Maybe #6 should be updated to list the files that need to move around and then we can watch the progress we make towards being all-the-way done with that.

I'm gonna leave a couple comments inline here, but apart from those I think this should get merged.

File: src/magic_folder/cli.py [hunk 1]
```diff
@@ -118,21 +118,21 @@
     read_config,
 )
 
-from ..frontends.magic_folder import (
+from magic_folder.frontends.magic_folder import (
```
Comment (exarkun) [inline src/magic_folder/cli.py]: Can this be `from .frontends...`?  I'm not dead set on this.  I don't think there's a clear winner between absolute and explicit relative imports in the general case.

File: src/magic_folder/cli.py [hunk 2]
```diff
@@ -118,21 +118,21 @@
...
     MagicFolder,
     load_magic_folders,
     save_magic_folders,
     maybe_upgrade_magic_folders,
 )
-from ..web.magic_folder import (
+from web.magic_folder import (
```
Comment (exarkun) [inline src/magic_folder/cli.py]: I think this definitely should be either `from .web...` or `from magic_folder.web...`.  *Implicit* relative imports are awful.

File: src/magic_folder/cli.py [hunk 3]
```diff
@@ -118,21 +118,21 @@
...
     magic_folder_web_service,
 )
 
-from ..status import (
+from status import (
```
Comment (exarkun) [inline src/magic_folder/cli.py]: Same comment here regarding implicit relative imports.

File: src/magic_folder/cli.py [hunk 4]
```diff
@@ -118,21 +118,21 @@
...
     status as _status,
 )
 
-from .._coverage import (
+from _coverage import (
```
Comment (exarkun) [inline src/magic_folder/cli.py]: And here

File: src/magic_folder/test/cli/test_magic_folder.py [hunk 5]
```diff
@@ -50,9 +50,8 @@
 from ...frontends.magic_folder import (
     MagicFolder,
 )
-from ...scripts import (
-    magic_folder_cli,
-)
+from magic_folder import cli as magic_folder_cli
```
Comment (exarkun) [inline src/magic_folder/test/cli/test_magic_folder.py]: explicit relative import here would be more consistent with surrounding code, not that I'm a huge fan of the dot counting that goes into understanding something like `from ...frontends.magic_folder import ...`

**Comment by meejah** [comment]
okay, i changed the imports so they're all relative. (And merged master)

**Pull request merged**